components/Table.tsx

```diff
@@ -265,6 +270,19 @@ export const Table: React.FC<Props> = ({ data, hasFilters = true }) => {
 
   const router = useRouter();
 
+  function onClickRow(row: TRow<Synth<{ type: ContractType }>>) {
+    if (row.original.externalUrl) {
+      const newWindow = window.open(
```

Comment:
nit: would doing this with an "a" element be better for SEO?

Reply:
I tend to agree with you, but how to achieve it? The whole row needs to be clickable. Write an `<a>` tag as a child in each `<Cell />` ?